CMakeLists.txt

```diff
@@ -435,12 +435,31 @@ add_custom_target(check-picolibc)
 # by add_subdirectory of llvm project
 set(LLVM_DEFAULT_EXTERNAL_LIT "${LLVM_BINARY_DIR}/bin/llvm-lit")
 
-function(get_qemu_command target_triple)
+function(get_test_executor target_triple qemu_machine qemu_cpu qemu_params)
     if(target_triple MATCHES "^aarch64")
-        set(qemu_command "qemu-system-aarch64" PARENT_SCOPE)
+        set(qemu_command "qemu-system-aarch64")
     else()
-        set(qemu_command "qemu-system-arm" PARENT_SCOPE)
+        set(qemu_command "qemu-system-arm")
     endif()
+
+    # picolibc tests
```

Comment:
I don't understand why this comment is in this function

Reply:
Good spot. I copied it over thoughtlessly. Done.